icons/ory/ory-original.svg

```diff
@@ -0,0 +1 @@
+<svg fill="#5528ff" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M 63.984375 13.164062 L 33.75 28.148438 C 33.71875 28.164062 33.679688 28.160156 33.648438 28.140625 C 33.617188 28.121094 33.597656 28.089844 33.601562 28.054688 L 33.601562 15.289062 C 33.601562 15.175781 33.652344 15.09375 33.753906 15.042969 L 63.902344 0.128906 C 64.066406 0.046875 64.257812 0.046875 64.421875 0.128906 C 74.378906 5.066406 84.363281 10.011719 94.378906 14.972656 C 94.941406 15.25 94.617188 15.417969 94.625 15.777344 C 94.675781 19.855469 94.671875 23.929688 94.609375 28.003906 C 94.609375 28.121094 94.554688 28.15625 94.445312 28.101562 L 64.296875 13.164062 C 64.191406 13.113281 64.089844 13.113281 63.984375 13.164062 Z M 63.984375 13.164062 "/><path d="M 94.675781 64 C 94.675781 80.863281 81.007812 94.53125 64.144531 94.53125 C 47.28125 94.53125 33.613281 80.863281 33.613281 64 C 33.613281 47.136719 47.28125 33.46875 64.144531 33.46875 C 81.007812 33.46875 94.675781 47.136719 94.675781 64 Z M 94.675781 64 "/><path d="M 64.199219 114.878906 L 94.503906 99.875 C 94.535156 99.859375 94.574219 99.859375 94.601562 99.878906 C 94.632812 99.898438 94.648438 99.929688 94.648438 99.964844 L 94.636719 112.839844 C 94.636719 112.875 94.617188 112.910156 94.582031 112.929688 L 64.171875 127.980469 C 64.144531 127.996094 64.113281 127.996094 64.082031 127.980469 L 33.65625 112.890625 C 33.621094 112.871094 33.597656 112.835938 33.601562 112.800781 L 33.601562 99.945312 C 33.601562 99.910156 33.617188 99.878906 33.648438 99.859375 C 33.675781 99.84375 33.714844 99.839844 33.742188 99.855469 L 64.109375 114.878906 C 64.136719 114.894531 64.171875 114.894531 64.199219 114.878906 Z M 64.199219 114.878906 "/></svg>
```

Comment:
Same goes here. A unified path is needed for all plain icons. Learn more here: https://github.com/devicons/devicon/wiki/SVG-Standards